Controller/MediaAdminController.php

```diff
@@ -77,10 +77,11 @@ public function listAction(Request $request = null)
         $datagrid->setValue('context', null, $context);
 
         // retrieve the main category for the tree view
-        $category = $this->container->get('sonata.classification.manager.category')->getRootCategory($context);
+        $rootCategory = $this->container->get('sonata.classification.manager.category')->getRootCategory($context); 
```

Comment:
In fact, you don't even need to highlight to see it : 

![sans titre](https://cloud.githubusercontent.com/assets/657779/21986250/fe97e700-dbfe-11e6-9ea9-4b3e712b595b.png)